index.html

```diff
@@ -1,13 +1,18 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Recipes</title>
 </head>
+
 <body>
   <h1> Odin Recipes </h1>
   <a href="./recipies/ScallopedPotato.html">Scalloped potato</a>
+  <a href="./recipies/CrabStuffedMushrooms.html">Crab Stuffed Mushrooms</a>
+  <a href="./recipies/chocolateChipCOokies.html">chocolate Chip Cookies</a>
```

Comment:
- [ ] The requirements tells to use an unordered list place each links on new line

![image](https://user-images.githubusercontent.com/49812651/147697932-118b1c6e-43e0-40ab-a1af-76c860bca79e.png)

- [ ]  The link to `chocolate Chip Cookies` is not working

![image](https://user-images.githubusercontent.com/49812651/147697940-d951d69b-0594-4396-a003-8f644de230da.png)